avm/res/insights/private-link-scope/main.bicep

```diff
@@ -242,49 +253,17 @@ output privateEndpoints array = [
     name: privateLinkScope_privateEndpoints[i].outputs.name
     resourceId: privateLinkScope_privateEndpoints[i].outputs.resourceId
     groupId: privateLinkScope_privateEndpoints[i].outputs.groupId
-    customDnsConfig: privateLinkScope_privateEndpoints[i].outputs.customDnsConfig
-    networkInterfaceIds: privateLinkScope_privateEndpoints[i].outputs.networkInterfaceIds
+    customDnsConfig: privateLinkScope_privateEndpoints[i].outputs.customDnsConfigs
+    networkInterfaceIds: privateLinkScope_privateEndpoints[i].outputs.networkInterfaceResourceIds
```

Comment:
Should ideally implement the output type for usability. Can be found, for example, here:
https://github.com/Azure/bicep-registry-modules/blob/d0a356d28654905f4505aac893d4ee767022a9d4/avm/res/purview/account/main.bicep#L592-L614

The output would then just also need to be changed to
```bicep
output privateEndpoints privateEndpointOutputType[] = [
```

Reply:
Note: This requires to also references the latest version of the PE module